### PR TITLE
Sal #1514 contextual carousel fix

### PR DESCRIPTION
**Pull request by szafarcin**

### CODE REVIEW

File: app/directives/carousel/carousel.directive.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var Carousel = function($timeout) {
     template: require("./carousel.template.jade"),
     link: function (scope, iElement, attrs, ctrl, transclude) {
       const FLOW_SIZE = 5;
-      const ITEM_WIDTH = 65;
+      const ITEM_WIDTH = scope.$root.appType === 'notifications' ? 65 : 60;
```
Comment (dominiksteiner) [inline app/directives/carousel/carousel.directive.js]: can you simply pass an optional itemWidth scope parameter into that directive?

**Comment by amirmehmood** [comment]
lgtm

File: app/styles/carousel.styl [hunk 2]
```diff
@@ -6,11 +6,17 @@
...
     &.loading
       display none
+    &.contextual
+      height 44px
   .carouselActive
     display block
     -webkit-perspective 200px
     height 52px
     width 100%
+    &.contextual .tab
```
Comment (dominiksteiner) [inline app/styles/carousel.styl]: actually let's move those styles into contextual.styl itself as the carousel is in hubUi and should have no notion of contextual but be reusable for other components too

File: app/styles/carousel.styl [hunk 3]
```diff
@@ -6,11 +6,17 @@
       display none
     &.loading
       display none
+    &.contextual
```
Comment (dominiksteiner) [inline app/styles/carousel.styl]: same here

**Comment by dominiksteiner** [comment]
lgtm, @albi34 please review

**Comment by albi34** [comment]
lgtm

**Pull request merged**